setup.cfg

```diff
@@ -22,7 +22,7 @@ author = Apache Software Foundation
 author_email = dev@airflow.apache.org
 url = https://airflow.apache.org/
 version = attr: airflow.__version__
-long_description = file: README.md
+long_description = file: generated/PYPI_README.md
```

Comment:
The generated folder is included in the `MANIFEST.in `https://github.com/apache/airflow/blob/main/MANIFEST.in#L42

So I am guessing this file will be available correctly and be used for the PyPI project listing description?
If someone can confirm my guess, would be great.

Reply:
You can build the package with `breeze release-management prepare-airflow-package --package-format sdist` and extract it to check if it’s in the package.

Reply:
Thank you.

Yes, confirmed that the file is included both in the `.tar.gz` and in the wheel's pkg description

<img width="857" alt="Screenshot 2023-08-30 at 2 44 28 PM" src="https://github.com/apache/airflow/assets/10206082/a833ded7-7a59-4cc4-883a-e8687fd8db33">